### PR TITLE
Internal: Container Fixes [ED-5484]

### PR DESCRIPTION
**Pull request by StyleShit**

### CODE REVIEW

File: assets/dev/js/editor/document/hooks/ui/delete/index.js [hunk 1]
```diff
@@ -2,3 +2,4 @@
 
 export { ColumnIsEmpty } from './column-is-empty';
 export { DeleteSectionIsFull } from './section-is-full';
```
Comment (iNewLegend) [inline assets/dev/js/editor/document/hooks/ui/delete/index.js:4]: Those two old hooks does not following the convention.
they need to be located at this path:
`document/elements/delete`.

File: assets/dev/js/editor/document/hooks/ui/delete/move-resizeable-handle.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
+import After from 'elementor-api/modules/hooks/ui/after';
```
Comment (iNewLegend) [inline assets/dev/js/editor/document/hooks/ui/delete/move-resizeable-handle.js:1]: It should be `assets/dev/js/editor/document/hooks/ui/document/elements/create/move-resizeable-handle.js`

Comment (StyleShit) [reply assets/dev/js/editor/document/hooks/ui/delete/move-resizeable-handle.js]: Done

File: includes/elements/container.php [hunk 3]
```diff
@@ -174,6 +178,38 @@ private function render_video_background() {
...
+	 * Render the Container's shape divider.
+	 * TODO: Copied from `section.php`.
+	 *
+	 * Used to generate the shape dividers HTML.
+	 *
+	 * @param string $side - Shape divider side, used to set the shape key.
+	 *
+	 * @return void
+	 */
+	private function render_shape_divider( $side ) {
```
Comment (iNewLegend) [inline includes/elements/container.php:191]: @StyleShit, suggestion: it can have common base.

**Review by iNewLegend — CHANGES_REQUESTED** [review]

File: includes/elements/container.php (hunk 3 above)
Comment (StyleShit) [reply includes/elements/container.php:191]: @matipojo 
Do we want to combine Container & Section shape divider controls using a Trait or something?

File: includes/elements/container.php [hunk 4]
```diff
@@ -195,14 +231,25 @@ private function print_html_tag() {
...
+?>
+		<?php
+
+		if ( $settings['shape_divider_top'] ) {
+			$this->render_shape_divider( 'top' );
+		}
+
+		if ( $settings['shape_divider_bottom'] ) {
+			$this->render_shape_divider( 'bottom' );
+		}
```
[the remark this answers is not in the document]
Comment (StyleShit) [reply includes/elements/container.php:252]: Done

File: assets/dev/scss/frontend/widgets/spacer.scss [hunk 5]
```diff
@@ -0,0 +1,10 @@
+.elementor-spacer-inner {
+	.e-container.e-r & {
```
Comment (matipojo) [inline assets/dev/scss/frontend/widgets/spacer.scss:2]: Use an understandable name for the class, here and below

File: includes/elements/container.php (hunk 3 above)
Comment (matipojo) [reply includes/elements/container.php:191]: No, we don't want to depend on each other

**Review by matipojo — CHANGES_REQUESTED** [review]

File: assets/dev/scss/frontend/widgets/spacer.scss (hunk 5 above)
Comment (StyleShit) [reply assets/dev/scss/frontend/widgets/spacer.scss:2]: Done

**Pull request merged**